.github/workflows/iroha2-dev-pr-title.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           regex: '^\[feature\] #\d+(, #\d+)*: .+$'
       # - name: Add feature label
       #   uses: actions-ecosystem/action-add-labels@v1
-      #   if: ${{ steps.feature-match.outputs.match != '' }}
+      #   if: steps.feature-match.outputs.match != ''
```

Comment:
I don't see why this change was necessary, but OK.

Reply:
I just intended to unify to either `if: expression` or `if: ${{ expression }}`

Reply:
Got it. Makes sense.